indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -588,6 +608,20 @@ public static Path makeSegmentOutputPath(
     return new Path(prependFSIfNullScheme(fileSystem, basePath), String.format("./%s", segmentDir));
   }
 
+  //This method is very similar to makeSegmentOutputPath() method except it returns output path upto version
```

Comment:
JobHelper isn't in the druid-api module and is not listed on http://druid.io/docs/latest/development/modules.html so I wouldn't consider it part of Druid's public API. So, it's fine to make changes to public methods.